test/fixtures/programmatic/plugin/broken-plugin/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
I have problems spotting the syntax error.

Maybe we can create file with obvious syntax error as e.g.:

```javascript
syntax
'error
```